app/controllers/item.js

```diff
@@ -31,134 +32,141 @@ var UserSourceAuth = require('app/models/userSourceAuth');
 var validateParams = require('app/lib/validateParams');
 
+var util= require('util');
+
 var queue = kue.createQueue();
 
-queue.process('storeItemData', function(queueJob, done) {
-  debug('process queueJob %s', queueJob.id);
+var tools = require('app/lib/utils/debuggingTools');
 
-  var getItem = (done) => {
-    Item.findById(queueJob.data.itemId, (error, item) => {
-      if (error) {
-        done(error);
-      } else if (!item) {
-        done(new Error('Item with queueJob.data.itemId not found'));
-      } else {
-        done(undefined, item);
-      }
-    });
-  };
+const STORE_ITEM_DATA = 'storeItemData';
+
+queue.process(STORE_ITEM_DATA, function(queueJob, done) {
+    console.log('process queueJob %s', queueJob.id);
+
+    var getItem = (done) => {
+        Item.findById(queueJob.data.itemId, (error, item) => {
+            if (error) {
+                done(error);
+            } else if (!item) {
+                done(new Error('Item with queueJob.data.itemId not found'));
+            } else {
+                done(undefined, item);
+            }
+        });
+    };
+
+    var getJob = (item, done) => {
+        Job.findById(queueJob.data.jobId, (error, job) => {
+            if (error) {
+                done(error);
+            } else if (!job) {
+                done(new Error('Job with queueJob.data.jobId not found'));
+            } else {
+                done(undefined, item, job);
+            }
+        });
+    };
+
+    var storeItemData = (item, job, done) => {
+        debug.start('queueJob storeItemData', item.id, job.id);
+        module.exports.storeItemData(item, queueJob.data.data, job, done);
+    };
+
+    async.waterfall([getItem, getJob, storeItemData], (error) => {
+        if (error) {
+            debug.error('failed to process queueJob %s: %s', queueJob.id, error.message);
+        } else {
+            debug.success('processed queueJob %s', queueJob.id);
+        }
 
-  var getJob = (item, done) => {
-    Job.findById(queueJob.data.jobId, (error, job) => {
-      if (error) {
         done(error);
-      } else if (!job) {
-        done(new Error('Job with queueJob.data.jobId not found'));
-      } else {
-        done(undefined, item, job);
-      }
     });
-  };
-
-  var storeItemData = (item, job, done) => {
-    debug.start('queueJob storeItemData', item.id, job.id);
-    module.exports.storeItemData(item, queueJob.data.data, job, done);
-  };
-
-  async.waterfall([getItem, getJob, storeItemData], (error) => {
-    if (error) {
-      debug.error('failed to process queueJob %s: %s', queueJob.id, error.message);
-    } else {
-      debug.success('processed queueJob %s', queueJob.id);
-    }
-
-    done(error);
-  });
 });
 
 queue.on('error', (error) => {
-  debug.error('queueJob failed:', error.message);
+    debug.error('queueJob failed:', error.message);
 });
 
 /**
  * Callback resource found at URL.
+ * (Get resource from passed URL and pass to callback (done)
```

Comment:
Was this comment intentional? In general they shouldn't start with a parenthesis like here.